packages/js-moi-providers/src/interaction.ts

```diff
@@ -31,12 +31,11 @@ const serializePayload = (ixType: IxType, payload: InteractionPayload): Uint8Arr
  * @param {Map<string, number | bigint>} values - The input Map with keys as hexadecimal strings.
  * @returns {Map<string, number | bigint>} - A new Map with trimmed keys.
  */
-const processValues = (values: Map<string, number | bigint>): Map<string, string> => {
-    const entries = new Map();
-
-    values.forEach((value, key) => entries.set(trimHexPrefix(key), toQuantity(value)))
-
-    return entries
+const processValues = (values: Map<string, number | bigint>): Record<string, string> => {
+    return Array.from(values).reduce((entries: Record<string, string>, [key, value]) => {
+        entries[key] = toQuantity(value);
+        return entries;
+    }, {});
```

Comment:
The return type mentioned in the comment doesn't match with the actual return value

Reply:
b2eb84c76d35164ed14f16d0312a9c3a86ead0dd